lib/repl.js

```diff
@@ -0,0 +1,64 @@
+var net = require("net");
+var repl = require("repl");
+var fs = require('fs');
+var path = require('path');
+var options = require('./args');
+var slice = require('slice-file');
+
+var ctrlC = new Buffer([ 255, 244, 255, 253, 6 ]).inspect();
+
+function bounce() {
+    var pidFile = path.join(options.tmp, 'registry-static.pid');
+    var self = this;
+    fs.readFile(pidFile, 'utf8', function(err, pid){
+        self.outputStream.write('Restarting process: '+pid+'\n');
+        process.kill(pid, 'SIGHUP');
+        self.displayPrompt();
+    });
+}
+
+var bounceDef = {
+    help: 'bounce the child process',
+    action: bounce
+};
+
+function liveLog() {
+    if (!options.log) {
+        this.outputStream.write("\nThe `log()` command only works when registry-static is invoked with a `--log` argument\n");
+        this.displayPrompt();
+    } else {
+        var tail = slice(options.log);
+        tail.follow(-10).pipe(this.outputStream);
+
+        // outputSteam on data?? yes. it's the socket.
+        this.outputStream.on('data', function dataListener(data){
+            // otherwise the repl gets into a weird state
+            if (ctrlC === data.inspect()) {
+                tail.close();
+            }
+        });
+    }
```

Comment:
This is ok for now, but we could hook into the actual logger and write to the repl and the logger at the same time. Then the check here doesn't matter.

https://github.com/davglass/registry-static/blob/master/lib/logger.js